service/src/main/java/uk/gov/hmcts/reform/fpl/model/HearingPreferences.java

```diff
@@ -12,12 +15,14 @@ public class HearingPreferences {
     private final String interpreter;
     private final String intermediary;
     private final String welshDetails;
```

Comment:
I guess adding to the other comment about these fields being deprecated, could annotate them as such in the model so we don't use them accidentally in the future